components/centraldashboard/app/server.ts

```diff
@@ -9,9 +9,10 @@ import {WorkgroupApi} from './api_workgroup';
 import {KubernetesService} from './k8s_service';
 import {getMetricsService} from './metrics_service_factory';
 
-const defaultKfam = process.env.NODE_ENV !== 'production'
-  ? 'localhost'
-  : 'profiles-kfam';
+const isProduction = process.env.NODE_ENV === 'production';
+const defaultKfam = isProduction
+? 'profiles-kfam.kubeflow'
```

Comment:
I don't know that you *need* to do this since the two services will be deployed together in the same namespace. @kunmingg suggested that on https://github.com/kubeflow/manifests/pull/298/files so if he's good with that, let's go for it. The only problem this could have is if somehow a different namespace was used for both services.

Reply:
I tried suggesting the same, but seems @jlewi also wants this change, so I think for now this is fine